packages/app-project/pages/[panoptesEnv]/[owner]/[project]/about/education.js

```diff
@@ -12,12 +12,12 @@ export async function getStaticProps({ locale, params }) {
   const pageTitle = page?.strings?.title ?? 'Education'
 
   return {
-    notFound,
```

Comment:
If you remove `notFound`, you’ll need to manually set the response code to 404. 

Otherwise, I think you’re serving a regular project page, with a 200 response, but the content says 404. Search engines will index that as a regular page, unless the status code is 404.

Reply:
You might be able to set the response code by throwing an error in the `App` component. I think that’s how we send 500 Server Error when Page components error.

Reply:
This took a lot of time to debug but the way NextJS operates is that returning the notFound value forces it to display only the 404 page... Which means no header bar, no current state, nothing being passed along. Therefore, to get app-like behavior (essentially the nav bar at the top) we have to return a 200 status code. To mitigate the search engines indexing the pseudo-404 page I've added a meta tag that tells robots (like search engines) to not index the page - which should result in no issues related to search engine indexing and SEO.